tests/unit/Codeception/Lib/GroupManagerTest.php

```diff
@@ -14,15 +14,22 @@ class GroupManagerTest extends \Codeception\Test\Unit
     // tests
     public function testGroupsFromArray()
     {
-        $this->manager = new GroupManager(['important' => ['UserTest.php:testName', 'PostTest.php']]);
-        $test1 = $this->makeTestCase('UserTest.php', 'testName');
-        $test2 = $this->makeTestCase('PostTest.php');
+        $this->manager = new GroupManager(['important' => ['tests/data/UserTest.php:testName', 'tests/data/PostTest.php']]);
+        $test1 = $this->makeTestCase('tests/data/UserTest.php', 'testName');
+        $test2 = $this->makeTestCase('tests/data/PostTest.php');
         $test3 = $this->makeTestCase('UserTest.php', 'testNot');
         $this->assertContains('important', $this->manager->groupsForTest($test1));
         $this->assertContains('important', $this->manager->groupsForTest($test2));
         $this->assertNotContains('important', $this->manager->groupsForTest($test3));
     }
 
+    public function testRealPathForFileWithMethodName()
+    {
+        $this->manager = new GroupManager(['important' => ['tests/data/PostTest.php:testName']]);
+        $test = $this->makeTestCase('tests/data/PostTest.php', 'testName');
```

Comment:
There is a lot of files in tests/data already, it would be a little bit cleaner to move these files to subdirectory, e.g. `tests/data/GroupManagerTest`

Reply:
I named it `group_manager_test` to be consistent with the other folder names on the same level